src/vasc/vasculature.cpp

```diff
@@ -36,28 +36,28 @@ Vasculature::Vasculature(const std::string& source) {
         throw UnknownFileType("File: " + source + " does not end with the .h5 extension");
     }
 
-    _properties = std::make_shared<property::Properties>(loader);
+    properties_ = std::make_shared<property::Properties>(loader);
 
-    buildConnectivity(_properties);
+    buildConnectivity(properties_);
 }
 
-Section Vasculature::section(const uint32_t& id) const {
-    return {id, _properties};
+Section Vasculature::section(uint32_t id) const {
+    return {id, properties_};
 }
 
 std::vector<Section> Vasculature::sections() const {
     std::vector<Section> sections_;
-    const auto& vasc_sections = _properties->get<property::VascSection>();
+    const auto& vasc_sections = properties_->get<property::VascSection>();
     sections_.reserve(vasc_sections.size());
-    for (size_t i = 0; i < vasc_sections.size(); ++i) {
-        sections_.emplace_back(static_cast<uint32_t>(i), _properties);
+    for (uint32_t i = 0; i < vasc_sections.size(); ++i) {
```

Comment:
should we worry about very large amount of `vasc_sections`?

Reply:
If we are going to support whole brain datasets, I think we should yeah. Should I replace with size_t or uint64_t ?

Reply:
`size_t` seems most natural to me